graph/typeDefs.graphql

```diff
@@ -865,6 +896,9 @@ type Settings {
 
   # domains will return a given list of domains.
   domains: Domains
+
+  # karma contains the currently set thresholds for triggering Trust beheviour.
+  karma: KarmaThresholds
```

Comment:
Maybe it'd be more descriptive to use `settings.karma.thresholds.flag.reliable`.
Right now accessing this we use `settings.karma.flag.reliable`, it wasn't immediately clear what this was until I saw the schema.

Reply:
We don't have any information to display relevant to the karma besides the thresholds, and as this is already the `settings` for the `karma`, I would assume it was implied that it would be relating to the settings for the the karma on flags.

Reply:
I'd like to make it easier to understand the code, without needing to lookup the graphql schema. Sth like `settings.karma.flagThresholds.reliable` would just crystal clear what we are accessing here.

Reply:
The issue with that solution is that we already have a graph endpoint: `user.reliable.flagger` and `user.reliable.flaggerKarma` which represents properties of their reliability, and their Karma. We could perhaps on the client, alias the field?